Rakefile

```diff
@@ -122,229 +145,18 @@ end
 
 def each_gem
   each_child do |dir|
-    gem_dir = dir.split('/').last.to_sym
-    if REGEN_METADATA.has_key?(gem_dir)
+    #puts "Each child #{dir}"
+    gem_dir = dir.split('/').last
```

Comment:
curious: why not use `.to_sym` while looking up in the dictionary ?

Reply:
we used to look for the symbol when we had the dictionary defined in the rakefile, now when loading the json file (config.json), these values are not loaded as symbols but strings. We can look into this further, if you think there could be benefits of having symbols.